test/run_tests.py

```diff
@@ -475,8 +477,9 @@ def run_command_line_test(
                 continue
 
             if ref_file.endswith(".json"):
-                output_json = json.loads(output_content)
-                ref_json = json.loads(ref_content)
+                # Empty strings are falsy. Fall back to an empty dict if the file is empty
+                output_json = json.loads(output_content) if output_content else dict()
```

Comment:
shouldn't we complain here about missing file?

Reply:
If we get here the file exists but is empty. If you do json.loads on an empty string it fails